### PR TITLE
Double and triple touch validation moved to the `delete_clock` function of the `on_touch_down` event

### PR DESCRIPTION
**Pull request by Neizvestnyj**
Double and triple click checks must occur when the `on_touch_down` method is triggered, otherwise the functions : `on_double_tap`, `on_triple_tap` - will trigger twice.
[Code example](https://kivymd.readthedocs.io/en/latest/behaviors/touch/index.html#usage)

### CODE REVIEW

File: kivymd/uix/behaviors/touch_behavior.py [hunk 1]
```diff
@@ -1,51 +1,34 @@
 """
 Behaviors/Touch
 ===============
-
```
Comment (HeaTTheatR) [inline kivymd/uix/behaviors/touch_behavior.py:4]: Why did you remove indentation from the documentation?

File: kivymd/uix/behaviors/touch_behavior.py [hunk 2]
```diff
@@ -1,51 +1,34 @@
...
     KV = '''
     Screen:
-
         MyButton:
             text: "PRESS ME"
             pos_hint: {"center_x": .5, "center_y": .5}
     '''
-
-
```
Comment (HeaTTheatR) [inline kivymd/uix/behaviors/touch_behavior.py:32]: The double indentation before the class declaration is the PEP8 standard.

**Comment by HeaTTheatR** [comment]
@Neizvestnyj Please attach some code examples that demonstrate the old behavior and behavior according to your change ...

File: kivymd/uix/behaviors/touch_behavior.py (hunk 2 above)
Comment (Neizvestnyj) [reply kivymd/uix/behaviors/touch_behavior.py:32]: `PyCharm` did not correct me), now I will record a video and create a new pool of request
